adapters/tappx/tappx.go

```diff
@@ -51,14 +51,58 @@ func (a *TappxAdapter) MakeRequests(request *openrtb2.BidRequest, reqInfo *adapt
 			Message: "Error parsing bidderExt object",
 		}}
 	}
-
 	var tappxExt openrtb_ext.ExtImpTappx
 	if err := json.Unmarshal(bidderExt.Bidder, &tappxExt); err != nil {
 		return nil, []error{&errortypes.BadInput{
 			Message: "Error parsing tappxExt parameters",
 		}}
 	}
 
+	//In this section we create all EXT parameters
+	type VideoExt struct {
+		Rewarded int `json:"rewarded,omitempty"`
+	}
+	if tappxExt.Rewarded == 1 {
+		request.Imp[0].Video.Ext = json.RawMessage(`{"rewarded": 1}`)
+	}
```

Comment:
I'm personally fine with this approach, but please understand this will overwrite all publisher provided ext values in the video object.

Reply:
Hi! A lot of thanks of this point. We now find useless to use the rewarded tappx param. We think that is better to tell to our publishers to send in video ext param directly like in all our oRTB connections. 😅

We clean all this in a2ddc1f commit.